src/smufl_maestro_wide_noteheads

```diff
@@ -0,0 +1,703 @@
+    The page preferences may be provided optionally to avoid loading them for each call.
+
+    @ system (FCStaffSystem)
+    @ [page_format_prefs] (FCPageFormatPrefs) page format preferences to use, if supplied.
+    : (boolean) `true` if the system was successfully updated.
+    ]]
+    function library.system_indent_set_to_prefs(system, page_format_prefs)
+        page_format_prefs = page_format_prefs or library.get_page_format_prefs()
+        local first_meas = finale.FCMeasure()
+        local is_first_system = (system.FirstMeasure == 1)
+        if (not is_first_system) and first_meas:Load(system.FirstMeasure) then
+            if first_meas.ShowFullNames then
+                is_first_system = true
+            end
+        end
+        if is_first_system and page_format_prefs.UseFirstSystemMargins then
+            system.LeftMargin = page_format_prefs.FirstSystemLeft
+        else
+            system.LeftMargin = page_format_prefs.SystemLeft
+        end
+        return system:Save()
+    end
+
+    --[[
+    % calc_script_name
+
+    Returns the running script name, with or without extension.
+
+    @ [include_extension] (boolean) Whether to include the file extension in the return value: `false` if omitted
+    : (string) The name of the current running script.
+    ]]
+    function library.calc_script_name(include_extension)
+        local fc_string = finale.FCString()
+        if finenv.RunningLuaFilePath then
+            -- Use finenv.RunningLuaFilePath() if available because it doesn't ever get overwritten when retaining state.
+            fc_string.LuaString = finenv.RunningLuaFilePath()
+        else
+            -- This code path is only taken by JW Lua (and very early versions of RGP Lua).
+            -- SetRunningLuaFilePath is not reliable when retaining state, so later versions use finenv.RunningLuaFilePath.
+            fc_string:SetRunningLuaFilePath()
+        end
+        local filename_string = finale.FCString()
+        fc_string:SplitToPathAndFile(nil, filename_string)
+        local retval = filename_string.LuaString
+        if not include_extension then
+            retval = retval:match("(.+)%..+")
+            if not retval or retval == "" then
+                retval = filename_string.LuaString
+            end
+        end
+        return retval
+    end
+
+    --[[
+    % get_default_music_font_name
+
+    Fetches the default music font from document options and processes the name into a usable format.
+
+    : (string) The name of the defalt music font.
+    ]]
+    function library.get_default_music_font_name()
+        local fontinfo = finale.FCFontInfo()
+        local default_music_font_name = finale.FCString()
+        if fontinfo:LoadFontPrefs(finale.FONTPREF_MUSIC) then
+            fontinfo:GetNameString(default_music_font_name)
+            return default_music_font_name.LuaString
+        end
+    end
+
+    return library
+
+end
+
```

Comment:
Could you delete everything above this line?

In this repository, we have a much of shared code. That way, we can reuse a ton of code across multiple scripts, speeding up the time it takes for us to write scripts and increasing the stability of our scripts (since we don't need to worry about every edge case every time).

You'll notice down below that Jacob imports our general library:

```lua
local library = require("library.general_library")
```

This is one of our files with shared code: https://github.com/finale-lua/lua-scripts/blob/master/src/library/general_library.lua

As you can see, this shared code is split across multiple files.

However, we don't want to make users download multiple files to use our scripts. Instead, we have it setup that when we update a script, all the used shared files are "bundled" into a single file. Everything above this line is the auto-generated code that bundles the general library into the script file. That's what users download for the website.

All that to say you can safely delete the above lines and everything will continue to work.

For reference, here's the original script Jacob wrote:

https://github.com/finale-lua/lua-scripts/blob/master/src/smufl_maestro_wide_noteheads.lua

And here's the bundled version that users download from the website:

https://github.com/finale-lua/lua-scripts/blob/master/dist/smufl_maestro_wide_noteheads.lua